packages/kit/src/vite/index.js

```diff
@@ -75,6 +75,9 @@ function kit() {
 	/** @type {import('vite').ConfigEnv} */
 	let vite_config_env;
 
+	/** @type {string[]} */
+	let allowed;
```

Comment:
do we need to store `allowed` separately when we already store `vite_config`? (although arguably we could store the `vite_config` in `configResolved`)

Reply:
right, but the only thing we use `vite_config` for is to get the `logLevel`. it's actually probably better for us to use the resolved config there, so if we switch `vite_config` to store the resolved config then we'd no longer need a separate `allowed` variable

Reply:
there's also `protocol`. the reason i didn't just save the whole thing is that there isn't a convenient type for the resolved config but yeah, we can save those three things inside `configResolved`

Reply:
oh! TIL. for some reason i didn't find that before — fixed